rfcs/079-new-online/README.md

```diff
@@ -0,0 +1,293 @@
+- 1 content provider (to curate the featured works, eg. Collection Information, Digital Production)
+
+The following stage is to integrate a digital item's `digitisedDate` and `digitisedVersion` into the catalogue-pipeline and API's Work model, as part of the item's [DigitalLocation](https://github.com/wellcomecollection/catalogue-pipeline/blob/53d04e063a75600236ac8ed41934b9c52b451624/common/internal_model/src/main/scala/weco/catalogue/internal_model/locations/Location.scala#L15C12-L15C27).  
+Colleagues required:
+- 1 BE developer (Platform)
+- 1 FE developer (Experience)
+
+
+## 1. New Prismic Custom Type ingested by the content pipeline and available through the content API
+
+Create an additional [ETL pipeline](https://github.com/wellcomecollection/content-api/blob/6e9c4ba7285c7ea1b259038a0ccb7ce9f6219da1/pipeline/src/extractTransformLoad.ts#L31) in the content pipeline, to load the new Prismic type ("`featured-work`"?) into an index in the content cluster.
+
+Potential `ElasticsearchFeaturedWork` type.  
+No aggreagations or filters required
+
+```typescript
+type ElasticsearchFeaturedWork = {
+  id: string; // work id 
+  display: FeaturedWork;
+  query: {
+    createdDate: string;
+    id: string
+    type: string[];
+    feature: string; // eg. "new-online", "trending"
+  };
+};
+
+type FeaturedWork = {
+  id: string; // work id 
+  url: string;  
+  title: string;
+  image: {
+    contentUrl: string;
+    width: number; // can we make this optional?
+    height: number; // can we make this optional?
+    alt?: string;
+  };
+  type: string[];
+  feature: string; // eg. "new-online", "trending"
+  meta?: string;
+}
+```
+
+On the API side the data is exposed on the `/featured/new-online` endpoint, with optional `to`, `from` and `type` query params.  
+Absent query params return `FeaturedWork[]` with the most recent of each of 4 default `types`, with feature="new-online".  
+Query params enable flexibility as to the date range and type of `ElasticsearchFeaturedWork` documents returned. 
+
+Pros: 
+- integrates into existing infrastructure
+- the Prismic Custom Type can be made to fit our needs
+- doesn't require code change and deployment to update the works
+- control over the types/labels
+- control over the cover image
+
+Cons: 
+- manually curated content risks getting stale 
+
+
+## 2. Integrate the digitised date and version in the Work model and the catalogue pipeline.
+
+Pros: 
+- No need for manual intervention, the New online items on the landing page will reflect what's actually been recently digitised.
+- `digitisedDate` in the Work model enables easier audit for Digital Production team.
+
+Cons: 
+- The digitisedDate will not always be strictly accurate: when a work is digitised again, the `CREATEDATE` is that of the latest digitisation. We can mitigate this by only returning documents where `digitisedVersion` is `v1`.
+- What do we do for work with advisory? 
+- Amending the Work model should not be done lightly 
```

Comment:
Could you add a short explanation of why? (I imagine it's partly because of "if we remove/modify it it means it's not backwards compatible") but there might be more to it and it's be good to outline.